mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -421,8 +421,12 @@ ConvolutionDims ConvolutionDims::fromOp(Operation *op) {
 
     if (filterAttr.getValue() == "y") {
       y = filterShape[i];
+    } else if (filterAttr.getValue() == "0") {
```

Comment:
Another place that accepts both old and new keys.  I thought I could just convert old forms to new at a few entry points, but then there are places that read conv ops from a file and there's no place to convert before they call functions like fromOp().  It surprised me how much of Operation construction is completely generic.